work_queue/src/work_queue.c

```diff
@@ -7543,6 +7549,25 @@ static void write_transaction_worker_resources(struct work_queue *q, struct work
 }
 
 
+static void write_transaction_transfer(struct work_queue *q, struct work_queue_worker *w, struct work_queue_task *t, size_t sizeinmb, int walltime, char* filename, int type){
+	struct buffer B;
+	buffer_init(&B);
+	buffer_printf(&B, "TRANSFER ");
+	if (type)
```

Comment:
Use braces for both the `if` and `else`, even it it is only one statement.
Also, now use
```
if(type == WORK_QUEUE_INPUT) {  
    ...
```